src/main/java/racingcar/Racing.java

```diff
@@ -4,24 +4,44 @@
 import java.util.List;
 
 public class Racing {
-    private List<Car> cars = new ArrayList<>();
+    private List<Car> readyCars = new ArrayList<>();
 
     public List<Car> getCars(){
-        return this.cars;
+        return this.readyCars;
     }
 
-    public void readyCar(int CarCount){
-        for(int i = 0; i < CarCount; i++){
-            cars.add(new Car());
+    public void readyCar(String[] cars){
+        for(int i = 0; i < cars.length; i++){
+            readyCars.add(new Car(cars[i], 0));
```

Comment:
```suggestion
        for (String car : cars) {
            readyCars.add(new Car(car, 0));
```
이렇게 변경하면 코드 읽기가 더 쉽겠네요😁